.gitmessage.txt

```diff
@@ -0,0 +1,32 @@
+<type>(<scope>): <subject>
```

Comment:
:red_circle: It is not required for this file to be added to version control. It only needs to be stored on the developers computer.